src/utils/filter.ts

```diff
@@ -63,3 +71,21 @@ export const makeCampRoomHitFilter = ({
 
   return isWithinBounds(checkPercent, roomBounds);
 };
+
+export const filterUnreadPrivateChats = (
+  chats: WithId<PrivateChatMessage>[],
+  user: FirebaseReducer.AuthState | undefined
+) => {
+  return (
+    chats
+      ?.filter(
+        (message) =>
+          message.from !== user?.uid &&
+          message.deleted !== true &&
+          message.type === "private" &&
+          message.ts_utc.seconds > HIDE_BEFORE &&
+          message.isRead === false
+      )
+      .sort(chatSort) ?? []
```

Comment:
[nit] You could either use a guard (eg. `if (!chats) return []`) up front, or possibly move the default value up front?

```suggestion
    if (!chats) return []
    
    return chats
      .filter(
        (message) =>
          message.from !== user?.uid &&
          message.deleted !== true &&
          message.type === "private" &&
          message.ts_utc.seconds > HIDE_BEFORE &&
          message.isRead === false
      )
      .sort(chatSort)
```

```suggestion
    (chats ?? [])
      .filter(
        (message) =>
          message.from !== user?.uid &&
          message.deleted !== true &&
          message.type === "private" &&
          message.ts_utc.seconds > HIDE_BEFORE &&
          message.isRead === false
      )
      .sort(chatSort)
```

Reply:
Updated.